.golangci.yml

```diff
@@ -0,0 +1,69 @@
+linters-settings:
+  govet:
+    check-shadowing: true
+  golint:
+    min-confidence: 0
+  maligned:
+    suggest-new: true
+  misspell:
+    locale: UK
+  gocritic:
+    enabled-tags:
+      - performance
+      - style
+      - experimental
+    disabled-checks:
+      - sloppyReassign
+      - docStub
+      - paramTypeCombine
+
+linters:
+  enable-all: true
+  disable:
+    - dupl
+    - gochecknoglobals
+    - gocyclo
+    - goconst
+    - lll
+    - misspell
+
+issues:
+  exclude-rules:
+    #Ignore variable scope linting
+    - path: "errors/error_handler_test.go"
+      linters:
+        - scopelint
+    - path: "config/config_test.go"
+      linters:
+        - scopelint
+    # Ignore things like "FormId should be FormID"
+    - path: codegen
+      text: "ST1003:"
+      linters:
+        - stylecheck
+    # We like underscores
```

Comment:
Really?

Reply:
haha I just copied this over from the existing .golangci.yml file in `server-lib`, so can't comment on the accuracy of liking underscores :)

Reply:
Removed this in the latest commits